src/components/document/resize-panel-divider.scss

```diff
@@ -38,10 +57,11 @@
       opacity: 0.01%;
       &:hover {
         opacity: 50%;
+      }
```

Comment:
Note that there was something odd with the braces in the previous implementation.